packages/web-react/src/components/Footer/demo/FooterMinimalistic.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Container } from '../../Container';
+import { Flex } from '../../Flex';
+import { Grid, GridItem } from '../../Grid';
+import { Link } from '../../Link';
+import { ProductLogo } from '../../ProductLogo';
+import { defaultSvgLogo } from '../../ProductLogo/demo/ProductLogoDefault';
+import { Select } from '../../Select';
+import Footer from '../Footer';
+
+const FooterMinimalistic = () => {
+  return (
+    <Footer paddingBottom="space-900" paddingTop="space-900">
+      <Container>
+        {/* Grid with product logo, social media links and language switch */}
+        <Grid
+          cols={{ mobile: 1, desktop: 12 }}
+          alignmentX={{ mobile: 'center', desktop: 'stretch' }}
+          alignmentY="center"
+          spacingY="space-1100"
+        >
+          {/* Product logo */}
+          <GridItem columnStart={{ desktop: 1 }} columnEnd={{ desktop: 4 }}>
+            <div className="text-desktop-left">
+              <Link href="https://www.example.com">
+                <ProductLogo>{defaultSvgLogo}</ProductLogo>
+              </Link>
+            </div>
+          </GridItem>
+
+          {/* Flex with secondary links */}
+          <GridItem columnStart={{ desktop: 4 }} columnEnd={{ desktop: 10 }}>
+            <nav className="text-center" aria-label="Secondary links">
```

Comment:
The alignment class should not be needed here, see the `web` demo.

```suggestion
            <nav aria-label="Secondary links">
```

Reply:
I put it there because of the mobile display. Without it, it is centered, but aligned to the left.

Without the class:
<img width="658" alt="image" src="https://github.com/user-attachments/assets/16e551bf-f615-4627-a8c9-4c83126e871e">

Reply:
With class:
<img width="497" alt="image" src="https://github.com/user-attachments/assets/f1ff9cee-cc8a-4998-ad1a-4121422386a7">